scalanet/src/io/iohk/scalanet/codec/FramingCodec.scala

```diff
@@ -84,4 +109,12 @@ object FramingCodec {
       m += 1
     }
   }
+
+  private def skipUnchecked(n: Int, from: ByteBuffer): Unit = {
+    var m: Int = 0
```

Comment:
you could use `.position(int)` method to move position in one operation, instead of incrementally

Reply:
Agreed. Changed.